### ISSUE TITLE
WARN SendHelper

### ISSUE THREAD

**Issue by jk20004** [AsamK/signal-cli#926]
Hello,
after month of working without any problem we see in the last days often the Error

`WARN SendHelper - Sending failed due to rate limiting from the signal server: [413] Rate limit exceeded: 413`

Any Idea?

**Comment by jk20004** [comment]
after updating Java (jdk-17.0.2) and signal-cli (signal-cli-0.10.4.2) i see 

```
Failed to send (some) messages:
+123456789: Network failure for "+123456789"

```
at 80% of the requests.

Can you please explain the error in more detail?

**Comment by AsamK** [comment]
Rate limiting is done by the Signal servers to protect from malicious users.
I don't know why this happens in your case. If you get a 413 response, the only thing you can do is retry later...

There's the possibility that signal-cli does something wrong to trigger too many network requests, so you could try running signal-cli with `--verbose --verbose` and post the logs here.

**Comment by AsamK** [comment]
error message fixed in c8f819cb9427fa58c81bdd02829aab97a94b7f33
